bundles/org.openhab.binding.omnilink/src/main/resources/OH-INF/thing/unit.xml

```diff
@@ -273,6 +281,10 @@
 		<label>Switch</label>
 		<description>Turn this room on/off.</description>
 		<category>Switch</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Light</tag>
+		</tags>
```

Comment:
As above comments, as I am confused why the category is not `lightbulb` if this channel is used for lights. Docs do not give enough info on what this channel does at a quick glance.

Reply:
The reasoning is that physically, the thing is a switch but the switch controls the light. The smart functionality is not in the bulb as it is in some cases but rather in the switch that controls the light. As such by following the strict definition of an openHAB Thing it made the most sense to make it a switch categorically.

Reply:
If <category>lightbulb</category> is used you get a light bulb icon that dynamically changes to reflect the state. But this is not important, all that needs to be considered is can it get used to switch a FAN on and off? If its use is only for lights, then I agree with the tags getting added. If the channel can be used for switching multiple different devices like a fan, then I suggest it should not be tagged in this way. As per the above comments, feel free to resolve this yourself if my concerns are wrong.

Reply:
Adding my two cents, I currently have ~53 UPB units controlled by the omnilink binding.  All are lights except for 2 bathroom fans.  The OmniPro board (and binding) treats them the same.  They are all UPB units and all show up as lights as default. I would prefer to go back and correct the 2 fans after discovery versus manually tagging all 53 for an odd or different device.

Reply:
That is my opinion as well, @swamiller.

Reply:
Happy with that so closing this.